pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroup.java

```diff
@@ -310,6 +336,21 @@ protected BytesAndMessagesCount updateLocalQuota(ResourceGroupMonitoringClass mo
         return oldBMCount;
     }
 
+    protected BytesAndMessagesCount getRgPublishRateLimiterValues() {
+        BytesAndMessagesCount retVal = new BytesAndMessagesCount();
+        final PerMonitoringClassFields monEntity =
+                                        this.monitoringClassFields[ResourceGroupMonitoringClass.Publish.ordinal()];
+        monEntity.localUsageStatsLock.lock();
+        try {
+            retVal.bytes = this.resourceGroupPublishLimiter.publishMaxByteRate;
```

Comment:
we can have a method that returns a BytesAndMessagesCount directly in `resourceGroupPublishLimiter`, this way we are not accessing directly the fields

Reply:
Thanks; fixed in iteration f925d4d